pipeline.py

```diff
@@ -51,19 +62,29 @@ class CorpusManager:
     """
 
     def __init__(self, path_to_raw_txt_data: str):
-        pass
+        self.path_to_raw_txt_data = path_to_raw_txt_data
+        self._storage = {}
+        self._scan_dataset()
 
     def _scan_dataset(self):
         """
         Register each dataset entry
         """
-        pass
+        path = Path(self.path_to_raw_txt_data)
+        for file in path.glob('*'):
+            file = file.name
+            for element in file:
+                if not element.isdigit():
```

Comment:
a cycle for checking a digit in a name (!) recall regex lecture from today and refactor